test/addons-napi/test_promise/test.js

```diff
@@ -38,23 +42,16 @@ promise.then(
   common.mustNotCall());
 test_promise.concludeCurrentPromise(Promise.resolve('chained answer'), true);
 
-assert.strictEqual(test_promise.isPromise(promise), true,
-                   'natively created promise is recognized as a promise');
+assert.strictEqual(test_promise.isPromise(promise), true);
 
-assert.strictEqual(test_promise.isPromise(Promise.reject(-1)), true,
-                   'Promise created with JS is recognized as a promise');
+assert.strictEqual(test_promise.isPromise(Promise.reject(-1)), true);
 
-assert.strictEqual(test_promise.isPromise(2.4), false,
-                   'Number is recognized as not a promise');
+assert.strictEqual(test_promise.isPromise(2.4), false);
 
-assert.strictEqual(test_promise.isPromise('I promise!'), false,
-                   'String is recognized as not a promise');
+assert.strictEqual(test_promise.isPromise('I promise!'), false);
 
-assert.strictEqual(test_promise.isPromise(undefined), false,
-                   'undefined is recognized as not a promise');
+assert.strictEqual(test_promise.isPromise(undefined), false);
 
-assert.strictEqual(test_promise.isPromise(null), false,
-                   'null is recognized as not a promise');
+assert.strictEqual(test_promise.isPromise(null), false);
 
-assert.strictEqual(test_promise.isPromise({}), false,
-                   'an object is recognized as not a promise');
+assert.strictEqual(test_promise.isPromise({}), false);
```

Comment:
Nit: I would prefer not to have single lines separated by empty lines for assertions.